core/src/solve.jl

```diff
@@ -521,19 +521,21 @@ function formulate!(
 
     for (id, rate) in zip(node_id, flow_rate)
         # Requirement: edge points away from the flow boundary
-        dst_id = only(outneighbors(graph_flow, id))
-
-        # Adding water is always possible
-        if rate >= 0
-            flow[id, dst_id] = rate
-        else
-            hasindex, basin_idx = id_index(basin.node_id, dst_id)
-            @assert hasindex "FlowBoundary intake not a Basin"
-
-            s = storage[basin_idx]
-            reduction_factor = min(s, 10.0) / 10.0
-            q = reduction_factor * rate
-            flow[id, dst_id] = q
+        # TODO: Check that only multiple outneighbours exist if these all go to fractionalflow
```

Comment:
If this is the final part of the issue, then I assume this TODO will stay.
Can you make an issue instead?